t/t-batch-transfer-size.sh

```diff
@@ -0,0 +1,97 @@
+﻿#!/usr/bin/env bash
```

Comment:
Thanks for adding these tests, this is great!

Other than one simplification, the only other thing I'd ask is if we can run:
```
$ git update-index --chmod=+x t/t-batch-transfer-size.sh 
```

and commit the result, so that the new file has the same executable permissions as the other `t/t-*.sh` files.  It's a minor detail, really, and makes no difference if you're using the `make t-foo.sh` approach to running the tests.  However, for those of us (like me!) who sometimes cheat a bit and just run `./t-foo.sh`, it helps to have the files all be executable.